streams/src/test/java/org/apache/kafka/streams/processor/internals/InternalTopicConfigTest.java

```diff
@@ -30,39 +30,65 @@
 
     @Test(expected = NullPointerException.class)
     public void shouldThrowIfNameIsNull() {
-        new RepartitionTopicConfig(null, Collections.<String, String>emptyMap());
+        new RepartitionTopicConfig(null, Collections.emptyMap());
     }
 
     @Test(expected = InvalidTopicException.class)
     public void shouldThrowIfNameIsInvalid() {
-        new RepartitionTopicConfig("foo bar baz", Collections.<String, String>emptyMap());
+        new RepartitionTopicConfig("foo bar baz", Collections.emptyMap());
+    }
+
+    @Test
+    public void shouldSetCreateTimeByDefault() {
```

Comment:
Seems we should test this for all 3 types of topics?

Reply:
Ack, added test for each internal topic type